src/utils/formatting.ts

```diff
@@ -40,7 +40,9 @@ export const addQueryParams = (url: string, queryParams: { [key: string]: string
     const value = queryParams[key];
 
     // null or undefined
-    if (value == null || !value?.length) return;
+    if (value == null) return;
```

Comment:
Was the double equals on purpose instead of triple?  Any reason not to just check truthy?

Reply:
It should check both `undefined` and `null`

Reply:
```suggestion
    if (!value) return;
```

Reply:
Then we will skip empty strings :) I will check, maybe we needed them for some cases.

Reply:
Yeah, for search playlists we get error for empty string:

`Search feeds require a search phrase`